### PR TITLE
Adjust helm/tiller installation instructions

### PR DESCRIPTION
**Pull request by MHBauer**
- tiller needs access later to install service-catalog objects

We could create an rbac role to bind to tiller specifically for the purposes of managing svc-cat resources installed using helm. We couldn't use helm to install that, so it would become a weird two step booting process. I think this ends up eventually solved by helm/tiller supporting impersonation. These instructions ought to be removed when impersonation support is gained.

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -32,17 +32,21 @@ be done with Helm setup.
...
 
-### RBAC Considerations
-
-If your kubernetes cluster has [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)
-enabled, you must ensure that the default service account for the `kube-system`
-namespace has the `cluster-admin` role:
+If your kubernetes cluster has
+[RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) enabled, you must
+ensure that tiller pod has `cluster-admin` access. The default install of tiller
+done with `helm init` installs the tiller pod into `kube-system` namespace and
```
Comment (nilebox) [inline docs/walkthrough.md]: > "install .... done ... with `helm init` installs the tiller pod into ... and uses ..."

doesn't sound grammatically correct to me, and it's also hard to get the meaning.
I would rephrase it to something like:
> By default, the `helm init` command installs the tiller pod into `kube-system` namespace, and uses the `default` service account.

Comment (kibbles-n-bytes) [inline docs/walkthrough.md]: I think grammatically it's fine, but @nilebox 's rephrasing is more clear so I'm also in favor of the change. Though I'd say no comma after "namespace".

@MHBauer I'd also think you dropped a "the" on line 37 "... ensure that tiller pod..."

**Comment by MHBauer** [comment]
@nilebox @kibbles-n-bytes thanks for review, I like suggestions. Please take another look.

**Comment by MHBauer** [comment]
Flake here. Merge commit of only doc changes failed.
